### PR TITLE
fix note delete to update according to code

### PR DESCRIPTION
**Pull request by lixiaobing1**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
According to code, update maybe better than delete.

**Which issue(s) this PR fixes**:
No

**Special notes for your reviewer**:
No

**Does this PR introduce a user-facing change?**:
No

```release-note
   NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:
No

### CODE REVIEW

**Review by damemi — COMMENTED** [review]
/lgtm
@lixiaobing1 could you please add a more descriptive title?
This also doesn't need a release note as it's not a user-facing change
/release-note none

File: pkg/scheduler/eventhandlers.go [hunk 1]
```diff
@@ -430,7 +430,7 @@ func addAllEventHandlers(
 		)
 	}
 
-	// On add and delete of PVs, it will affect equivalence cache items
+	// On add and update of PVs, it will affect equivalence cache items
```
Comment (Huang-Wei) [inline pkg/scheduler/eventhandlers.go:433]: Equivalence cache has been removed, so could you help remove those wordings?

```suggestion
	// On add and update of PVs.
```

**Comment by lixiaobing1** [comment]
> /lgtm
> @lixiaobing1 could you please add a more descriptive title?
> This also doesn't need a release note as it's not a user-facing change
> /release-note none

@damemi, ok has been modified.

File: pkg/scheduler/eventhandlers.go (hunk 1 above)
Comment (lixiaobing1) [reply pkg/scheduler/eventhandlers.go]: ok, has removed the note.

Comment (Huang-Wei) [reply pkg/scheduler/eventhandlers.go]: Yes, please update the comment to be "// On add and update of PVs."

Comment (Huang-Wei) [reply pkg/scheduler/eventhandlers.go]: And please squash the commits along with the new edit.

File: pkg/scheduler/eventhandlers.go [hunk 2]
```diff
@@ -430,8 +430,7 @@ func addAllEventHandlers(
 		)
 	}
 
-	// On add and delete of PVs, it will affect equivalence cache items
-	// related to persistent volume
+	// On add and delete of PVs
```
Comment (Huang-Wei) [inline pkg/scheduler/eventhandlers.go:433]:
```suggestion
	// On add and delete of PVs.
```

Comment (Huang-Wei) [reply pkg/scheduler/eventhandlers.go:433]: Can you squash the commits?

Comment (Huang-Wei) [inline pkg/scheduler/eventhandlers.go:433]:
```suggestion
	// On add and update of PVs.
```

Comment (lixiaobing1) [reply pkg/scheduler/eventhandlers.go]: ok.

**Review by damemi — APPROVED** [review]
/lgtm

File: pkg/scheduler/eventhandlers.go [hunk 3]
```diff
@@ -430,8 +430,7 @@ func addAllEventHandlers(
 		)
 	}
 
-	// On add and delete of PVs, it will affect equivalence cache items
-	// related to persistent volume
+	// On add and delete of PVs.
```
Comment (Huang-Wei) [inline pkg/scheduler/eventhandlers.go:433]: Isn't it update?

Comment (damemi) [reply pkg/scheduler/eventhandlers.go:433]: Oh right, got confused sorry
/lgtm cancel

Comment (lixiaobing1) [reply pkg/scheduler/eventhandlers.go:433]: It has been updated.

Comment (damemi) [reply pkg/scheduler/eventhandlers.go:433]: @lixiaobing1 this line still says `On add and delete of PVs`, it should say `On add and update of PVs`. Sorry for the confusion

Comment (lixiaobing1) [reply pkg/scheduler/eventhandlers.go]: It's my mistake, has been modify to: On add and update of PVs.

**Pull request merged**